lib/openstudio/workflow/adapters/input/local.rb

```diff
@@ -308,6 +308,23 @@ def epjson(user_options, default, logger)
           return default
         end
 
+        def ft_options(user_options, default, logger)
+
+          # check version for this feature
+          os_version = OpenStudio::VersionString.new(OpenStudio::openStudioVersion())
+          min_version_feature = OpenStudio::VersionString.new("3.3.0")
+          unless os_version >= min_version_feature
+            log_message = "ft_options subhash are only supported for versions >= 3.3.0. Setting to empty by default"
+            logger.info log_message
+            return default
+          end
+
+          # user option trumps all others
+          return user_options[:ft_options] if user_options[:ft_options]
+
+          return default
+        end
+
```

Comment:
I didn't bother trying to backport it. But technically most of the flags have been there for a while. If you want I could do that.

here are the timelines on the flags:

```
All OS Versions:
    :runcontrolspecialdays => :setKeepRunControlSpecialDays,
    :ip_tabular_output => :setIPTabularOutput,
    :no_lifecyclecosts => :setExcludeLCCObjects,

Since OS 3.0.0:
    :no_sqlite_output => :setExcludeSQliteOutputReport,
    :no_html_output => :setExcludeHTMLOutputReport,
    :no_variable_dictionary => :setExcludeVariableDictionary,

Will be new in 3.3.0:
    :no_space_translation => :setExcludeSpaceTranslation,
```

Reply:
I did an alternative that does handle backporting in f04f069deefc7f96a90471b525e3f5627524c657